CONTRIBUTING.md

```diff
@@ -29,6 +29,35 @@ VIP_PROXY="" API_HOST=http://localhost:4000 node ./dist/bin/vip app
 VIP_PROXY="" API_HOST=http://localhost:4000 node ./dist/bin/vip -- wp option get home
 ```
 
+### Using debugger
+
+Who doesn't like a good console.log for debugging?
+Well, sometimes it's insufficient, luckily it's not too complicated to use a debugger.
+
+1. First, make sure to run the `npm run build:watch`, this will generate source maps
+1. (Optional) run `npm link`
+1. Run the command you want via `node --inspect`, like so:  `node --inspect /opt/homebrew/lib/node_modules/@automattic/vip/dist/bin/vip-dev-env-import-sql.js`
```

Comment:
confusing to say `npm link` is optional but then target the linked path. i would document it using the local path. `npm link` seems like a totally separate decision that has no effect on debugging

```suggestion
1. Run the command you want via `node --inspect`, like so:  `node --inspect ./dist/bin/vip-dev-env-import-sql.js`
```